File: 1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums, target):
        dict_1={}
        for i in range(0,len(nums)):
            n=nums[i]
            result=target-n
            if result in dict_1:
                return[dict_1[result],i]
            dict_1[n]=i
        return []

File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int a=0;
        int b=0;
        for(int i=0;i<nums.length-1;i++){
            int sum=0;
            for(int j=i+1;j<nums.length;j++){
                sum=nums[i]+nums[j];
                if(sum==target){
                    a=i;
                    b=j;
                }
                else
                   sum=sum-nums[j];
                    
            }
        }
        return new int[]{a,b};
    }
}